project-management/src/main/java/life/qbic/projectmanagement/domain/model/measurement/NGSIndex.java

```diff
@@ -0,0 +1,19 @@
+package life.qbic.projectmanagement.domain.model.measurement;
+
+/**
+ * <b><record short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+public record NGSIndex(String indexI5, String indexI7) {
+
+  public static NGSIndex doubleIndexing(String indexI5, String indexI7) {
```

Comment:
It might make sense to be a litte bit more explicit here and create subclasses `record NGSDoubleIndex` and `record NGSSingleIndex`

Reply:
While i like the idea, i would defer the splitting up into singular indices to once we better understand on the different use cases of indices in the NGS domain 🤔